examples/life.js

```diff
@@ -0,0 +1,119 @@
+qKit.init({border:{width:8,color:'#824222'}});
+
+const suffusion	=192;
+
+var humor=new Array(2);
+humor[0]=new Array(9);
+humor[1]=new Array(9);
+
+//was dead
+humor[0][0]='#000000';
+humor[0][1]='#001840';
+humor[0][2]='#003050';
+humor[0][3]='#40FF60';//born
+humor[0][4]='#202840';
+humor[0][5]='#20283C';
+humor[0][6]='#1C2438';
+humor[0][7]='#182030';
+humor[0][8]='#101820';
+
+//was alive
+humor[1][0]='#602010';
+humor[1][1]='#702018';
+humor[1][2]='#D0FF00';//lives
+humor[1][3]='#F8FF60';//lives
+humor[1][4]='#803830';
+humor[1][5]='#44342C';
+humor[1][6]='#38302C';
+humor[1][7]='#302820';
+humor[1][8]='#281818';

```

Comment:
Not sure if you needed it, but in case you need random color you can use `qKit.util.randomColor()`

Reply:
Nope.  These are judiciously chosen colors.